packages/altair-core/src/script/evaluator-worker-engine.ts

```
@@ -112,4 +106,21 @@ export class ScriptEvaluatorWorkerEngine {
     });
   }
+
+  private async evalWithContext(
+    script: string,
+    context: {
+      altair: GlobalHelperContext;
+      alert: (msg: string) => Promise<Promise<void>>;
+    }
+  ) {
+    const contextEntries = Object.entries(context);
+    const fn = (0, eval)(`
```

Biome (2.4.16)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)